sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateProperties.cs

```diff
@@ -89,34 +89,31 @@ public CertificateProperties(Uri id)
         public bool? Enabled { get => _attributes.Enabled; set => _attributes.Enabled = value; }
 
         /// <summary>
-        /// Gets or sets not before date in UTC.
+        /// Gets a <see cref="DateTimeOffset"/> of when the certificate will be valid.
         /// </summary>
         public DateTimeOffset? NotBefore { get => _attributes.NotBefore; internal set => _attributes.NotBefore = value; }
 
         /// <summary>
-        /// Gets or sets expiry date in UTC.
+        /// Gets a <see cref="DateTimeOffset"/> of when the certificate will expire.
         /// </summary>
         public DateTimeOffset? ExpiresOn { get => _attributes.ExpiresOn; internal set => _attributes.ExpiresOn = value; }
 
         /// <summary>
-        /// Gets creation time in UTC.
+        /// Gets a <see cref="DateTimeOffset"/> of when the certificate was created.
         /// </summary>
         public DateTimeOffset? CreatedOn { get => _attributes.CreatedOn; internal set => _attributes.CreatedOn = value; }
 
         /// <summary>
-        /// Gets last updated time in UTC.
+        /// Gets a <see cref="DateTimeOffset"/> of when the certificate was updated.
         /// </summary>
         public DateTimeOffset? UpdatedOn { get => _attributes.UpdatedOn; internal set => _attributes.UpdatedOn = value; }
 
         /// <summary>
-        /// Gets reflects the deletion recovery level currently in effect for
-        /// secrets in the current vault. If it contains 'Purgeable', the
-        /// secret can be permanently deleted by a privileged user; otherwise,
-        /// only the system can purge the secret, at the end of the retention
-        /// interval. Possible values include: 'Purgeable',
-        /// 'Recoverable+Purgeable', 'Recoverable',
-        /// 'Recoverable+ProtectedSubscription'
+        /// Gets the recovery level currently in effect for certificates in the Key Vault.
+        /// If "Purgeable", the certificates can be permanently deleted by an authorized user;
```

Comment:
Technically, the comma goes inside quotation marks.  Should this be code formatted instead?

Reply:
IIRC, that was only when quoting someone or showing speech. Used to emphasize a defined term, I don't believe that is the case. That said, quotes probably aren't appropriate to emphasize terms. Instead, perhaps I should use `<c></c>` blocks here and everywhere else I used this same statement. The reason I didn't format it as a table is because, as a summary, IntelliSense won't show it formatted. I don't think it even supports tables or lists.